apps/meteor/app/mail-messages/server/functions/sendMail.ts

```diff
@@ -9,7 +9,7 @@ import { placeholders } from '../../../utils/server';
 import { SystemLogger } from '../../../../server/lib/logger/system';
 import * as Mailer from '../../../mailer';
 
-export const sendMail = function (from: string, subject: string, body: string, dryrun: boolean, query: string): void {
+export const sendMail = function (from: string, subject: string, body: string, dryrun?: boolean, query?: string): void {
```

Comment:
We've the opportunity here to transform this huge signature into a object as syntatic sugar for named parameters on call.